drivers/bbram/bbram_npcx.c

```diff
@@ -82,8 +84,9 @@ static int bbram_npcx_write(const struct device *dev, size_t offset, size_t size
 {
 	const struct bbram_npcx_config *config = dev->config;
 
+	printk("offset=%zu, size=%zu, max_size=%zu\n", offset, size, config->size);
```

Comment:
How about using `LOG_DBG` to replace `printk`?

Reply:
Oh man, good catch, I actually thought I removed all these. Thank you!